cmd/cli/install.go

```diff
@@ -148,17 +150,24 @@ func newInstallCmd(config *helm.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVar(&inst.enableMetricsStack, "enable-metrics-stack", true, "Enable metrics (Prometheus and Grafana) deployment")
 	f.StringVar(&inst.meshName, "mesh-name", defaultMeshName, "name for the new control plane instance")
 	f.BoolVar(&inst.deployJaeger, "deploy-jaeger", true, "Deploy Jaeger in the namespace of the OSM controller")
+	f.StringArrayVar(&inst.setOptions, "set", []string{}, "set values on the command line (can specify multiple or separate values with commas: key1=val1, key2=val2)")
```

Comment:
> `key1=val1, key2=val2`

I'm not certain a space between these will work. Not a big deal if it doesn't, but I would take that out of the help message just to make it a bit clearer. I see the help message for Helm's `--set` doesn't include the space.

Reply:
thanks.. taken care of..